pkg/cli/admin/release/info.go

```diff
@@ -692,11 +692,12 @@ type ReleaseManifestDiff struct {
 }
 
 type ReleaseInfo struct {
-	Image          string                          `json:"image"`
-	ImageRef       imagesource.TypedImageReference `json:"-"`
-	Digest         digest.Digest                   `json:"digest"`
-	ContentDigest  digest.Digest                   `json:"contentDigest"`
-	IsManifestList bool
+	Image              string                          `json:"image"`
+	ImageRef           imagesource.TypedImageReference `json:"-"`
+	Digest             digest.Digest                   `json:"digest"`
+	ContentDigest      digest.Digest                   `json:"contentDigest"`
+	IsManifestList     bool
+	ManifestListDigest string
 	// TODO: return the list digest in the future
 	// ListDigest    digest.Digest                       `json:"listDigest"`
```

Comment:
@Prashanth684 is this TODO comment related to the same work you implemented here?

Reply:
This is a good point

Reply:
ooh...yeah it does look like that. @soltysh was this field meant to capture the manifestlist digest. if so i can just use it.

Reply:
removed TODO and made the manifestlistdigest a digest.Digest type